docs/connector-development/config-based/low-code-cdk-overview.md

```diff
@@ -145,5 +145,5 @@ This section a tutorial that will guide you through the end-to-end process of im
 For examples of production-ready config-based connectors, refer to:
 
 - [Greenhouse](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-greenhouse)
-- [Sendgrid](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml)
-- [Sentry](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-sentry/source_sentry/sentry.yaml)
+- [Sendgrid](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-sendgrid/source_sendgrid)
```

Comment:
```suggestion
- [Sendgrid](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-sendgrid/source_sendgrid/manifest.yaml)
```

Reply:
Was following the source-greenhouse link scheme. I updated all 3 to manifest.yaml